library/src/main/java/com/pokegoapi/api/settings/Settings.java

```diff
@@ -123,6 +123,23 @@ public void updateSettings(DownloadSettingsResponse response) {
 		if (response.getSettings().hasGpsSettings()) {
 			gpsSettings.update(response.getSettings().getGpsSettings());
 		}
+		// check if new api has been forced by Niantic
+		if(!response.getSettings().getMinimumClientVersion().isEmpty()) {
+			int[] currentVer = api.getHashProvider().getCurrentAPIVersion();
+			String[] curVer = response.getSettings().getMinimumClientVersion().split("\\.");
```

Comment:
`curVer` implies that this is the current version of the API, and this conflicts with the above `currentVer`. You then use `minimumVer` for the same values as ints. Maybe use `splitMinimumVer` instead?